package.xml

```diff
@@ -25,19 +30,32 @@
   <build_depend>random_numbers</build_depend>
   <build_depend>resource_retriever</build_depend>
   <build_depend>shape_msgs</build_depend>
+  <build_depend>geometry_msgs</build_depend>
   <build_depend>visualization_msgs</build_depend>
+  <build_depend>rclcpp</build_depend>
+
+  <exec_depend>rclcpp</exec_depend>
```

Comment:
Please merge `<build_depend>` and `<exec_depend>` tags into `<depend>`.

Reply:
Makes sense. Isn't that something we want for melodic-devel as well?

Reply:
Sure.